doc/manual/src/language/values.md

```diff
@@ -92,39 +92,48 @@
 
 - <a id="type-path" href="#type-path">Path</a>
 
-  *Paths*, e.g., `/bin/sh` or `./builder.sh`. A path must contain at
-  least one slash to be recognised as such. For instance, `builder.sh`
-  is not a path: it's parsed as an expression that selects the
-  attribute `sh` from the variable `builder`. If the file name is
-  relative, i.e., if it does not begin with a slash, it is made
-  absolute at parse time relative to the directory of the Nix
-  expression that contained it. For instance, if a Nix expression in
-  `/foo/bar/bla.nix` refers to `../xyzzy/fnord.nix`, the absolute path
-  is `/foo/xyzzy/fnord.nix`.
-
-  If the first component of a path is a `~`, it is interpreted as if
-  the rest of the path were relative to the user's home directory.
-  e.g. `~/foo` would be equivalent to `/home/edolstra/foo` for a user
-  whose home directory is `/home/edolstra`.
-
-  For instance, evaluating `"${./foo.txt}"` will cause `foo.txt` in the current directory to be copied into the Nix store and result in the string `"/nix/store/<hash>-foo.txt"`.
-
-  Note that the Nix language assumes that all input files will remain _unchanged_ while  evaluating a Nix expression.
+  *Paths* are distinct from strings and can be expressed by path literals such as `./builder.sh`.
+
+  Paths are the preferred type for referring to local files.
+  This is thanks to the following properties:
+  - Path values are always in a canonical form, so that you are relieved from trailing slashes, `.` and `..`.
```

Comment:
> refrain from making usage suggestions

This is a good inclination, which I try to follow. I have diverged, because this needs to differentiate paths from strings, and normally I would just describe the purpose instead of the usage, but the purpose of path values is too broad; including absolute paths, which are usually bad.

> value judgements

Accidental pun? :)

> merely state interface (in this case, syntax) and behavior.

Canonicity is a crucial property of all path value behavior.
This page is about both strings and values, it'd be a failure not to differentiate them by this crucial property.

I'll make the implied alternative explicit: strings.

How about this?

```suggestion
  Paths are suitable for referring to local files, and are often preferable over strings.
  - Path values do not contain trailing slashes, `.` and `..`, as they are resolved when evaluating a path literal.
```

I know the first sentence still reads as a subjective statement, but I don't know how else to say it.

Reply:
This is good!